src/configuration/services/mongodb.md

```diff
@@ -108,3 +108,12 @@ The port may vary in your case.  Then, simply connect to that port locally using
 ```bash
 mongodump --port 30000
 ```
+
+Depending on the actual mongodb setup it can be required to pass additional parameters:
```

Comment:
MongoDB should be capitalized.

I'm curious, what additional setup conditions would require the full authenticated version of the command?